Config.uk

```diff
@@ -0,0 +1,17 @@
+menuconfig LIBLITTLECMS
+    bool "LittleCMS - color management engine"
+    depends on HAVE_LIBC
+    default n
+
+if LIBLITTLECMS
+    config LIBLITTLECMS_FASTFLOAT
+    bool "Fast Float plugin (licensed GPL3)"
+    select LIBINTEL_INTRINSICS if ARCH_X86_64
+    default n
+
+    config LIBLITTLECMS_THREAD
+    bool "Multi-threading plugin (licensed GPL3)"
+    depends on LIBMUSL_THREAD
```

Comment:
Is it possible to have something more generic here regarding threading support?

Reply:
I think removing the multi-threading support is not a bad idea, I especially don't like the licensing thing.

> newlibc also provides some pthread support, but it's not comprehensive enough to have this plugin build

Would `lib-pthread-embedded` do? That's what we used with `newlibc` before `musl`.